addons/sourcemod/scripting/include/shop/items.inc

```diff
@@ -543,16 +543,3 @@ stock ItemId Shop_GetArrayItem(ArrayList array, int index)
 // | || || || || |	| || || || || |	
 // \ /\ /\ /\ /\ /	\ /\ /\ /\ /\ /	
 //  V  V  V  V  V	 V  V  V  V  V	
```

Comment:
Maybe drop this too? I didn't see something like this in another includes.

Reply:
removed